### DISCUSSION TITLE
Choose SSH and database name.

### DISCUSSION THREAD

**Discussion by donniewr** [andreapollastri/cipi#376] [Ideas]
Hello,

I think CIPI should allow us to choose the SSH username and database name/password when we create a new site on a server.

This will make it easier when moving websites between servers. 

Currently everything is generated and if you move a site to another CIPI server you need to update everything (config.php etc) since all the usernames, databases and path have changed.

It can also be confusing when you manage many websites on the same server and all the login details looks the same.

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
